### ISSUE TITLE
Character won't show items to OR from other characters

### ISSUE THREAD

**Issue by aezea** [tullamods/Bagnon#1346]
I'm sure I am just missing a setting somewhere, but I can't figure it out.... <feels dumb>
My MC cannot see inventory of my other char, and my other char cannot see the inventory of my MC. BUT all the other char can see each others, it's like my MC is on another planet...
I have uninstalled/reinstalled, reloaded ui, etc etc. What am I missing!?

**Comment by Galthariel** [comment]
same issue

**Comment by aezea** [comment]
Several people I talked to asked me about BagBrother, which I could not find on my Addons List. However, after I reinstalled Bagnon, it came up on the character with the problem (I didn't notice if it was on the other characters). So, it's above all the other Bagnon addon check boxes. Make sure it is checked and you should be able to see other char and they should see you. If that doesn't fix it, it will be something else, but that was what fixed mine.

TLDR:
Reinstall Bagnon
Look for Bagbrother in Addons menu
Make sure Bagbrother is checked
Reload UI or restart WOW
![image](https://user-images.githubusercontent.com/94136104/141830497-857c696b-b722-4f7a-98fc-a584c0d89570.png)

**Comment by FireReaver** [comment]
It's definitely not only that, because some few days ago BagBrother worked for me.
Not anymore now, so blizz broke some APIs again.
This is the error I get when clicking on my character icon:

Message: ...\Wildpants\libs\Sushi-3.1\classes\drops\Dropdown.lua:136: Invalid data provided for `:SetBackdrop`
Time: Mon Nov 15 22:27:20 2021
Count: 1
Stack: ...\Wildpants\libs\Sushi-3.1\classes\drops\Dropdown.lua:136: Invalid data provided for `:SetBackdrop`
[string "=[C]"]: in function `assert'
[string "@Interface\AddOns\Bagnon\common\Wildpants\libs\Sushi-3.1\classes\drops\Dropdown.lua"]:136: in function `SetBackdrop'
[string "@Interface\AddOns\Bagnon\common\Wildpants\libs\Sushi-3.1\classes\drops\Dropdown.lua"]:48: in function `New'
[string "@Interface\AddOns\Bagnon\common\Wildpants\libs\Sushi-3.1\classes\drops\Dropdown.lua"]:57: in function `Toggle'
[string "@Interface\AddOns\Bagnon\common\Wildpants\classes\ownerSelector.lua"]:32: in function <...ns\Bagnon\common\Wildpants\classes\ownerSelector.lua:30>

Locals: (*temporary) = false
(*temporary) = "Invalid data provided for `:SetBackdrop`"

**Comment by FireReaver** [comment]
UPDATE: I can confirm that Hyphie24 fix does here work flawlessly: https://github.com/tullamods/Bagnon/issues/1345#issuecomment-966021613
